org.damask_multiphysics.DAMASK_grid.yaml

```diff
@@ -0,0 +1,64 @@
+id: org.damask_multiphysics.DAMASK_grid
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: DAMASK_grid
+finish-args:
+  - --filesystem=xdg-documents
+
+modules:
+  - name: petsc_damask_grid
+    buildsystem: simple
+    build-options:
+      env:
+        PETSC_ARCH: "gfortran-openmpi"
+        PETSC_DIR: "/run/build/petsc_damask_grid"
+    build-commands:
+      - ./configure --with-shared-libraries=1 --with-mpi-f90module-visibility=0
+        --prefix=/app --with-packages-download-dir=/run/build/petsc_damask_grid
+        --download-openmpi=1 --download-fblaslapack=1 --download-hdf5=1 --with-hdf5-fortran-bindings=1 --download-fftw=1
+        && make all install
+    sources:
+      - type: archive
+        url: https://web.cels.anl.gov/projects/petsc/download/release-snapshots/petsc-3.21.4.tar.gz
+        sha256: a9ae076d4617c7d84ce2bed37194022319c19f19b3930edf148b2bc8ecf2248d
+      - type: file
+        url: https://download.open-mpi.org/release/open-mpi/v5.0/openmpi-5.0.3.tar.gz
+        sha256: 3e12160dd1d6f5cd8897bbab63a05fb9223a1e20a1726db23607aac25dc1aa19
+      - type: file
+        url: https://bitbucket.org/petsc/pkg-fblaslapack/get/v3.4.2-p3.tar.gz
+        sha256: 644629f3559bf7ecdb8d01975131f982b15da823a39c5df4fa5764a4a5e9b34a
+      - type: file
+        url: https://web.cels.anl.gov/projects/petsc/download/externalpackages/hdf5-1.14.3-p1.tar.bz2
+        sha256: ec0da2dd048b4933603e810e1a341ba497a1e0b464dc3118549f38307945ff6c
+      - type: file
+        url: https://www.fftw.org/fftw-3.3.10.tar.gz
+        sha256: 56c932549852cddcfafdab3820b0200c7742675be92179e59e6215b340e26467
+
+  - name: libfyaml_damask_grid
+    buildsystem: autotools
+    build-commands:
+      - ./bootstrap.sh && ./configure --prefix=/app && make && make install
+    sources:
+      - type: archive
+        url: https://github.com/pantoniou/libfyaml/releases/download/v0.9/libfyaml-0.9.tar.gz
+        sha256: 7731edc5dfcc345d5c5c9f6ce597133991a689dabede393cd77bae89b327cd6d
+
+  - name: damask_grid
+    buildsystem: cmake
```

Comment:
this should be cmake-ninja it's faster.

Reply:
done

Reply:
Doesn't seem changed

Reply:
missed that, sorry. ninja does not work (also when trying to compile manually). It seems that preprocessor statements in Fortran confuse ninja and I get errors like:

```bash
[70/139] Building Fortran object src/CMakeFiles/DAMASK_grid.dir/prec.f90.o
FAILED: src/CMakeFiles/DAMASK_grid.dir/prec.f90.o src/prec.mod 
/usr/bin/mpif90 -I/home/m/DAMASK/src -fopenmp -std=f2018 -pedantic-errors -O2 -mtune=native  -cpp -fPIE -ffree-line-length-none -fimplicit-none -Wall -Wextra -Wcharacter-truncation -Wunderflow -Wsuggest-attribute=pure -Wsuggest-attribute=noreturn -Wconversion-extra -Wimplicit-procedure -Wunused-parameter -ffpe-summary=all -I/opt/petsc/linux-c-opt/include -I/usr/include/suitesparse -I/usr/include -I/usr/include/superlu  -Jsrc -fpreprocessed -c src/CMakeFiles/DAMASK_grid.dir/prec.f90-pp.f90 -o src/CMakeFiles/DAMASK_grid.dir/prec.f90.o
/home/m/DAMASK/src/prec.f90:21:11:

   21 |   ! https://stevelionel.com/drfortran/2017/03/27/doctor-fortran-in-it-takes-all-kinds
      |           1
Error: C++ style comments are not allowed in ISO C90
/home/m/DAMASK/src/prec.f90:21:11: note: (this will be reported only once per input file)
[72/139] Building Fortran object src/CMakeFiles/DAMASK_grid.dir/quit.f90.o
ninja: build stopped: subcommand failed.
make: *** [Makefile:13: grid] Error 1
```

Reply:
unfortunate :(

Reply:
yes. I'll open an issue upstream.